web/src/components/job/JobList.js

```diff
@@ -14,7 +14,7 @@ const TimeStampCell = (params) => {
   return params.value
     ? new Date(params.value).toLocaleDateString('en-AU') +
         ' ' +
-        new Date(params.value).toLocaleTimeString('en-AU', {hour12: false, hour: '2-digit', minute: '2-digit'})
+        new Date(params.value).toLocaleTimeString('en-AU', {hour: 'numeric', minute: '2-digit'})
```

Comment:
Since it's only used for toLocaleTimeString I don't think it's necessary.

Reply:
So you mean this setting will not be use in other area of the app? I am a bit concern on date formatting mismatch in different parts of the apps in the future plus if you have a test case that target correct formatting you may need a global setting that share with test and code.

Reply:
For the moment Jobs are the only entities that refer to a locally formatted timestamp.

Reply:
@[gsatimos](https://github.com/gsatimos)  I am not sure if it is over engineering or not as I am new to the system. Your view?

Reply:
Both good points @nspool and @utas-raymondng . I'm leaning toward @nspool 's argument that since it's currently contained within 1 line of code (okay, several lines but logically 1 line) in this 1 file then I would leave it as it is. 

Not saying I don't agree with @utas-raymondng 's point either though. If it were me though I'd leave as is. 

I guess technically you could bring in something like https://www.npmjs.com/package/locale-enum but then why bring in an extra package.

Reply:
Cool, than approve the PR